chain-core/src/init/network.rs

```diff
@@ -1,5 +1,82 @@
-#[derive(Debug)]
+use std::sync::Once;
+static INIT_NETWORK: Once = Once::new();
+static INIT_NETWORK_ID: Once = Once::new();
```

Comment:
I'm thinking whether there's a need for two "Once" -- it makes things more flexible, so perhaps yes

Reply:
ok